apps/web/components/dashboard/bookmarks/BookmarkOptions.tsx

```diff
@@ -118,6 +112,11 @@ export default function BookmarkOptions({ bookmark }: { bookmark: ZBookmark }) {
         setOpen={setTextEditorOpen}
       />
       <DropdownMenu>
+        <DeleteBookmarkConfirmationDialog
+          bookmark={bookmark}
+          open={deleteBookmarkDialogOpen}
+          setOpen={setDeleteBookmarkDialogOpen}
+        />
```

Comment:
I think this probably should be moved out of the dropdown menu. Maybe move it beside the other modals on line 108?

Reply:
[add delete bookmark description to i18n](https://github.com/hoarder-app/hoarder/pull/787/commits/52924c63425199140a4fba762518a2a8c01d1fc4)